src/templates/static/metatrader/download.jsx

```diff
@@ -10,99 +28,180 @@ const DownloadBlock = ({
     os,
     target,
     title,
+    children,
 }) => (
-    <div className='gr-4 gr-6-m gr-padding-10'>
-        <img className='gr-6 gr-centered' src={it.url_for(`images/pages/metatrader/icons/${image}.svg`)} />
+    <div className='gr-4 gr-12-m gr-padding-10'>
+        <img
+            className='gr-12 gr-centered mt-mobile-phone'
+            src={/[.]/.test(image) ? it.url_for(`images/pages/metatrader/icons/${image}`) : it.url_for(`images/pages/metatrader/icons/${image}.svg`)}
```

Comment:
is it only about having `.svg` at the end or not? if so we can move the ternary inside so we don't repeat the path, what do you think?